### ISSUE TITLE
V1.7 XamlParseException on new project.

### ISSUE THREAD

**Issue by rfuzzo** [ElinamLLC/SharpVectors#160]
Hello, 

On a new project (Net 5.0), I'm getting consistently a System.Windows.Markup.XamlParseException when using SharpVectors and SharpVectors Reloaded v1.7, when using a SvgViewBox.

I am not getting the error on v1.6 of .Reloaded.

Thank you. 

Minimal example: 

```
<Window x:Class="WpfApp3.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:WpfApp3"
        xmlns:svgc="http://sharpvectors.codeplex.com/svgc/"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800">
    <Grid>
        <svgc:SvgViewbox Source="https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/rg1024_green_grapes.svg" 
                         HorizontalAlignment="Center" VerticalAlignment="Center"/>
    </Grid>
</Window>
```

Csproj:
```
<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net5.0-windows</TargetFramework>
    <UseWPF>true</UseWPF>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="SharpVectors.Reloaded" Version="1.7.0" />
  </ItemGroup>
</Project>
```

Stacktrace
```System.Windows.Markup.XamlParseException
  HResult=0x80131501
  Message='Set property 'SharpVectors.Converters.SvgViewbox.Source' threw an exception.' Line number '12' and line position '10'.
  Source=PresentationFramework
  StackTrace:
   at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadComponent(Object component, Uri resourceLocator)
   at WpfApp3.MainWindow.InitializeComponent() in C:\Users\ghost\source\repos\WpfApp3\WpfApp3\MainWindow.xaml:line 1
  This exception was originally thrown at this call stack:
    SharpVectors.Runtime.NativeMethods.GetWindowHwnd(System.Windows.Media.Visual)
    SharpVectors.Runtime.DpiUtilities.GetWindowHwnd(System.Windows.Media.Visual)
    SharpVectors.Runtime.DpiUtilities.GetWindowScale(System.Windows.Media.Visual)
    SharpVectors.Converters.SvgViewbox.CreateDrawing(System.Uri, SharpVectors.Renderers.Wpf.WpfDrawingSettings)
    SharpVectors.Converters.SvgViewbox.CreateDrawing()
Inner Exception 1:
SvgErrorException: Object reference not set to an instance of an object.
Inner Exception 2:
NullReferenceException: Object reference not set to an instance of an object.```

**Comment by paulushub** [comment]
@rfuzzo Sorry, the code for DPI `HwndSource.FromVisual(visual)` is failing.

**Comment by paulushub** [comment]
Sorry, the fix is now active, Version 1.7.1.

**Comment by rfuzzo** [comment]
@paulushub Grand! Thank you very much for the quick fix. Can confirm it works. 
Your package is amazing!
